.github/workflows/localize-site.yml

```
@@ -0,0 +1,86 @@
+      - name: hreflang consistency audit
+        run: node scripts/check-hreflang.js
+
+      - name: Commit changes (if any)
+        run: |
+          if [ -n "$(git status --porcelain)" ]; then
+            git config user.name "wts-localize-bot"
+            git config user.email "actions@users.noreply.github.com"
+            git add th la fr en sitemap.xml 2>/dev/null || true
+            git commit -m "Regenerate localized pages from published translations
+
+Triggered: ${{ github.event.inputs.reason || 'manual run' }}"
```

actionlint (1.7.12)

[error] unexpected key "Triggered" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)